packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -51,6 +51,10 @@ cbp-list {
       padding-block: var(--cbp-list-item-padding-block);
     }
 
+    cbp-icon{
+      vertical-align: text-top;
```

Comment:
I would look into applying this to the icon component itself and test across different patterns. It makes sense and will be the right choice anywhere text line-height comes into play.

I tested a little and didn't see where it had much effect except on cards. That is because you are overriding the display prop, which I don't think you should.